modules/apps/roles/roles-admin-api/src/main/java/com/liferay/roles/admin/role/type/contributor/RoleTypeContributor.java

```diff
@@ -109,6 +109,10 @@ public default String getClassName() {
 	 */
 	public int getType();
 
+	public default String getTypeLabel() {
+		return getName();
+	}
```

Comment:
Would it be reasonable to instead use `RoleConstants.getTypeLabel(getType())` as the default? If we did that, we wouldn't have to update any of implementations.